apps/client/src/pages/WrapperPage.js

```diff
@@ -1,141 +1,64 @@
-import { Button, Heading, Paragraph } from "@datapunt/asc-ui";
-import React, { useContext } from "react";
+import React, { useState } from "react";
```

Comment:
I thought we were going to call this CheckerPage because WrapperPage was for Questions only. But maybe we should discuss the naming, so leave it is for now.